src/instrumentation/handlers/tealium-track.ts

```diff
@@ -0,0 +1,23 @@
+import {EventHandler, InstrumentationEvent} from '../types';
+
+export interface ExtendedWindow extends Window {
+  tealiumTrack(e: InstrumentationEvent): void;
+}
+
+const extendedWindow: ExtendedWindow | null =
+  typeof window !== 'undefined' ? ((window as Window) as ExtendedWindow) : null;
+
+export function sendEventTrackingToTealium(e: InstrumentationEvent) {
+  if (extendedWindow) {
```

Comment:
Maybe we can check for 
```suggestion
  if (extendedWindow && extendedWindow.tealiumTrack) {
```